### DISCUSSION TITLE
Gratification minimale de stage : calcul du nombre de jours ouvrés dans un mois

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2642] [Idées]
- [ ] La gratification des stagiaires se base sur un nombre d'heures effectifs, et non moyennés comme pour les salaires. Il oscille entre 126 et 154 selon le mois (mai avec jours fériés, vs janvier). 

> - Pour calculer le montant de la gratification, l'organisme d'accueil doit décompter le nombre d'heures de présence effective que vous avez effectuées durant votre stage.
> - Pendant un congé de maternité, de paternité ou d'adoption, ou autres autorisations d'absence, prévus à la convention, la gratification des jours d'absence n'est pas obligatoire.
> - Le montant de la gratification doit figurer dans votre convention de stage.
> - Le taux horaire de la gratification est égal au minimum à 3,9 € par heure de stage, correspondant à 15 % du plafond horaire de la sécurité sociale (soit 26 € x 0,15).

*[source](https://www.service-public.fr/particuliers/vosdroits/F32131)*

A voir également, le simulateur de la DILA : https://www.service-public.fr/simulateur/calcul/gratification-stagiaire

**Comment by mquandalle** [comment]
Pas sûr de bien comprendre, le plafond de la sécu est proratisé pour le calcul des cotisations ? ou bien c'est la gratification minimale qui est définie comme 15% du PASS (déjà pris en compte) ?

**Comment by johangirod** [comment]
> Pas sûr de bien comprendre, le plafond de la sécu est proratisé pour le calcul des cotisations ? ou bien c'est la gratification minimale qui est définie comme 15% du PASS (déjà pris en compte) ?

Ah, oui, exact, je me suis emmêlé les pinceaux, j'ai vu 15% PASS j'ai cru que c'était pour le calcul des cotisations :oups:

**Comment by mquandalle** [comment]
Je ferme en faveur de  #475
